src/js/public/support.js

```diff
@@ -1,95 +1,127 @@
-            productModelError.textContent = 'Campo Obligatorio';
-          }
-          if (serialNumber.value.trim() === '') {
-            serialNumberError.classList.add('support-form__error--active');
-            serialNumberError.textContent = 'Campo Obligatorio';
-          }
-    } else {
-        supportForm.style.transform = 'translateX(-200%)';
-        progressName[currentFieldset - 1].classList.add('support-progress__name--active');
-        progressBullet[currentFieldset - 1].classList.add('support-progress__bullet--active');
-        currentFieldset += 1;
+    function handleProgress(step) {
+        progressName[step - 1].classList.add("support-progress__name--active");
+        progressBullet[step - 1].classList.add("support-progress__bullet--active");
     }
-});
-firstPrevBtn.addEventListener('click', function() {
-    supportForm.style.transform = 'translateX(0%)';
-    progressName[currentFieldset - 2].classList.remove('support-progress__name--active');
-    progressBullet[currentFieldset - 2].classList.remove('support-progress__bullet--active');
-    currentFieldset -= 1;
-});
 
-// FIELDSET 3
-const secondPrevBtn = document.querySelector('#btn-prev-2');
-const selectCategories = document.querySelector('#support-problem-type');
-const problemMessage = document.querySelector('#support-problem');
+    const commonCustomViewErrors = {
+        add: function (field, message, cls) {
+            updateErrorContainer(field, message, cls);
+        },
+        remove: function (field, cls) {
+            updateErrorContainer(field, "", cls);
+        },
+    };
 
-secondPrevBtn.addEventListener('click', function() {
-    supportForm.style.transform = 'translateX(-100%)';
-    progressName[currentFieldset - 2].classList.remove('support-progress__name--active');
-    progressBullet[currentFieldset - 2].classList.remove('support-progress__bullet--active');
-    currentFieldset -= 1;
-});
-supportForm.addEventListener('submit', function(e) {
-    const selectCategoriesError = selectCategories.closest('.support-form__field').querySelector('.support-form__error');
-    const problemMessageError = problemMessage.closest('.support-form__field').querySelector('.support-form__error');
-    selectCategoriesError.classList.remove('support-form__error--active');
-    problemMessageError.classList.remove('support-form__error--active');
-    selectCategoriesError.textContent = '';
-    problemMessageError.textContent = '';
+    const configSteps = {
+        container: ".support-form__step",
+        button: ".next-step",
+        validationBy: "onclick",
+        selectors: SELECTORS,
+        messages: MESSAGES,
+        customViewErrors: commonCustomViewErrors,
+        onContainerSuccess: function (container) {
+            currentStep++;
+            supportForm.style.transform = `translateX(-${(currentStep - 1) * 100}%)`;
+            fieldsetsElements[currentStep - 1].classList.add("support-form__fieldset--active");
```

Comment:
```suggestion
            fieldsetsElements.forEach((fieldset) => {
               fieldset.classList.remove("support-form__fieldset--active");
            });
            fieldsetsElements[currentStep - 1].classList.add("support-form__fieldset--active");
```
Es importante remover el modificador --active de los fieldsets que ya no se estan mostrando